src/modules/map/components/WarningRibbon.vue

```diff
@@ -14,7 +14,7 @@ export default {
 
 <style lang="scss" scoped>
 @import 'src/scss/media-query.mixin';
-@import 'src/scss/variables-admin';
+@import 'src/scss/variables-admin.module';
```

Comment:
Why the `.module` here ?

Reply:
so that Vite handles this SCSS file as something it can send to Javascript (that's what I read somewhere in Vite support/documentation)
Without this module, it won't export things to JS